From 5ba16037184126f4ecd1b0b477598fb3e91b0e36 Mon Sep 17 00:00:00 2001
From: Keyon Jie <yang.jie@linux.intel.com>
Date: Wed, 20 Jun 2018 16:50:51 +0800
Subject: [PATCH 1707/2345] ASoC: Intel: bxt-tdf8532: change probe and trace
 buffer dai_links to dynamic

We should use .dynamic for all FE dai_links, so change probe and trace
buffer ones here to align to that.

Signed-off-by: Keyon Jie <yang.jie@linux.intel.com>
---
 sound/soc/intel/boards/bxt_tdf8532.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sound/soc/intel/boards/bxt_tdf8532.c b/sound/soc/intel/boards/bxt_tdf8532.c
index ffd9cc5..1e0e633 100644
--- a/sound/soc/intel/boards/bxt_tdf8532.c
+++ b/sound/soc/intel/boards/bxt_tdf8532.c
@@ -266,6 +266,7 @@ static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.nonatomic = 1,
+		.dynamic = 1,
 	},
 	{
 		.name = "Bxt Compress Probe capture",
@@ -276,6 +277,7 @@ static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.nonatomic = 1,
+		.dynamic = 1,
 	},
 	/* Trace Buffer DAI links */
 	{
@@ -287,6 +289,7 @@ static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
 		.platform_name = "0000:00:0e.0",
 		.capture_only = true,
 		.ignore_suspend = 1,
+		.dynamic = 1,
 	},
 	{
 		.name = "Bxt Trace Buffer1",
@@ -297,6 +300,7 @@ static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
 		.platform_name = "0000:00:0e.0",
 		.capture_only = true,
 		.ignore_suspend = 1,
+		.dynamic = 1,
 	},
 	/* Back End DAI links */
 	{
-- 
2.7.4

